paddle/scripts/paddle_build.bat

```diff
@@ -73,6 +73,9 @@ if %ERRORLEVEL% EQU 0 (
     git branch last_pr
 )
 
+:: set CI_SKIP_TEST if only *.py changed
+git diff --name-only %BRANCH% | findstr /V "\.py" || set CI_SKIP_TEST=ON
```

Comment:
can be named CI_SKIP_CPP_TEST, avoid misunderstanding

Reply:
done